python/lsst/daf/persistence/test/testObject.py

```diff
@@ -44,18 +44,35 @@ class TestObjectPair(object):
     """An object for testing that contains 2 objects.
     """
 
-    def __init__(self, objA, objB):
+    def __init__(self, objA=None, objB=None):
```

Comment:
(This really belongs in the other PR) I'm not sure you have complete test coverage of all the cases: generic assembler/disassembler with constructor, "_" setter, capitalized setter, and non-generic policy-specified assembler/disassembler